dashboard/app/models/lessons_resource.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: lessons_resources
+#
+#  lesson_id   :integer          not null
+#  resource_id :integer          not null
+#
+# Indexes
+#
+#  index_lessons_resources_on_lesson_id_and_resource_id  (lesson_id,resource_id) UNIQUE
+#  index_lessons_resources_on_resource_id_and_lesson_id  (resource_id,lesson_id)
```

Comment:
Not particularly import but why is this index not unique?

Reply:
this is sort of arbitrary. both indexes are needed in order to perform efficient queries, but only one needs to be unique to guarantee uniqueness.